core/src/test/java/org/elasticsearch/search/query/SimpleQueryStringTests.java

```diff
@@ -60,6 +61,13 @@ public void testSimpleQueryString() throws ExecutionException, InterruptedExcept
         assertSearchHits(searchResponse, "1", "2", "3");
 
         searchResponse = client().prepareSearch().setQuery(
+                boolQuery()
+                    .should(simpleQueryStringQuery("foo").boost(-10.0f))
```

Comment:
Small suggestion: maybe add some explanation or another test case that shows why its the negative boost that  makes the difference in the first hit result. Makes reading the test a little easier.